vhost-device-console/src/backend.rs

```diff
@@ -182,6 +192,7 @@ mod tests {
             backend: BackendType::Nested,
             tcp_port: String::from("12345"),
             socket_count: 1,
+            max_queue_size: 128,
```

Comment:
What about defining a constant here like you did in the other tests defining `DEFAULT_QUEUE_SIZE` ?

Reply:
Fixed.